Removed All Possible Bloats

loannaflip opened the pull request:
In this commit I've tried to remove all bloats as much as possible. Also I've rearranged pretty much everything in a good order. Overall code is more standard and high quality than before.

loannaflip committed:
Removed All Possible Bloats

Uniminin commented:
@JesterOrNot @Kreyren Ping!!!

Kreyren commented:
@Uniminin Did you make any benchmarks for this?

Kreyren commented:
lgtm excluding results from benches

JesterOrNot commented on gui/gui.py:
```diff
@@ -1,44 +1,49 @@
+        [sg.T(" " * 3), sg.Submit("BUILD"), sg.Cancel()]
+    ]
+
+    mainWindow = sg.Window('Docker Freeze Gui', default_element_size=(40, 1)).Layout(layout)
```
2 things,
1. Dockerfreeze is all one word
2. GUI is all caps

JesterOrNot reviewed (changes requested):
Thank you! :tada:
Just a few suggestions

loannaflip committed:
rearranged the name

Uniminin commented:
@JesterOrNot There you go. You're most welcome.

loannaflip committed:
added Main Function

The main function is added to prevent (certain) code from being run when the module is imported

JesterOrNot commented:
LGTM
